pkg/scheduler/factory/plugins.go

```diff
@@ -370,7 +371,14 @@ func buildScoringFunctionShapeFromRequestedToCapacityRatioArguments(arguments *s
 	if err != nil {
 		klog.Fatalf("invalid RequestedToCapacityRatioPriority arguments: %s", err.Error())
 	}
-	return shape
+	resources := make(map[v1.ResourceName]int64, 0)
+	for _, resource := range arguments.Resources {
+		resources[resource.Name] = int64(resource.Weight)
```

Comment:
how to handle a case in which two resources' name are duplicated in config and later in `RequestedToCapacityRatioArguments`?

Reply:
As we are storing this in a map the key will be one and the first value for the weight would be overwritten by the later.